diff --git a/Makefile.am b/Makefile.am
index 08a7bf5..5302093 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -13,7 +13,7 @@ LOCALEDIR_CPPFLAGS = -DLOCALEDIR='"$(localedir)"'
 PYTHON_CPPFLAGS = -I/usr/include/python$(PYTHON_VERSION)
 
 ## Targets
-SUBDIRS = po docs
+SUBDIRS = po 
 TESTS = tests/config_test.sh tests/files_test tests/pwhash_test tests/utils_test
 if LDAP
 TESTS += tests/ldap_test
diff --git a/configure.in b/configure.in
index e8e80e3..f67d091 100644
--- a/configure.in
+++ b/configure.in
@@ -144,6 +144,6 @@ GTK_DOC_CHECK
 dnl Does not even compile
 AM_CONDITIONAL([KRB5], [false])
 
-AC_CONFIG_FILES([Makefile po/Makefile.in docs/Makefile docs/reference/Makefile
+AC_CONFIG_FILES([Makefile po/Makefile.in 
 libuser.pc])
 AC_OUTPUT
diff --git a/docs/Makefile.am b/docs/Makefile.am
index 1bc84ea..5843ec5 100644
--- a/docs/Makefile.am
+++ b/docs/Makefile.am
@@ -6,16 +6,8 @@ CLEANFILES = $(man_MANS) sgml/libuser.html sgml/libuser-1.html \
 	sgml/libuser-2.html sgml/libuser.txt
 EXTRA_DIST = attributes.txt libuser.conf.5.in rfc2307.txt sgml/libuser.sgml
 
-all: sgml/libuser.txt sgml/libuser.html
 
 libuser.conf.5: $(srcdir)/libuser.conf.5.in Makefile
 	sed 's,@sysconfdir\@,$(sysconfdir),g' \
 		< $(srcdir)/libuser.conf.5.in > $@
 
-sgml/libuser.txt: $(srcdir)/sgml/libuser.sgml
-	[ -d sgml ] || mkdir sgml
-	cd sgml; sgml2txt ../$(srcdir)/sgml/libuser.sgml
-
-sgml/libuser.html: $(srcdir)/sgml/libuser.sgml
-	[ -d sgml ] || mkdir sgml
-	cd sgml; sgml2html ../$(srcdir)/sgml/libuser.sgml
